tests/testProjectedLikelihood.py

```diff
@@ -191,8 +192,7 @@ def testDirect(self):
         ctrl = lsst.meas.modelfit.UnitTransformedLikelihoodControl()
         var = numpy.random.rand(self.bbox0.getHeight(), self.bbox0.getWidth()) + 2.0
         self.exposure0.getMaskedImage().getVariance().getArray()[:, :] = var
-        efv = lsst.meas.modelfit.EpochFootprintVector()
-        efv.push_back(lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0))
+        efv = [lsst.meas.modelfit.EpochFootprint(self.footprint0, self.exposure0, self.psf0)]
```

Comment:
Fair enough. I discussed this with @jdswinbank yesterday. We should do a performance regression test at some point (probably after merge) and identify potential pain points.
If it turns out we need something in between that looks more like exposing `vector` as a view then we can always do so. But if we can get around doing that I think that would be a win.